app/src/ui/stash-changes/stash-and-switch-branch-dialog.tsx

```diff
@@ -94,6 +95,10 @@ export class StashAndSwitchBranch extends React.Component<
   }
 
   private renderStashActions() {
+    if (!enableStashing()) {
```

Comment:
:art: I don't think this is required, because the two places where we launch this popup are already guarded by the feature flag:

https://github.com/desktop/desktop/blob/8015f2dc412a075428ccce5b412dd80ed172d50d/app/src/lib/stores/app-store.ts#L2781-L2795

https://github.com/desktop/desktop/blob/8015f2dc412a075428ccce5b412dd80ed172d50d/app/src/lib/stores/app-store.ts#L2849-L2863

Feel free to tidy this up to keep the diff readable

Reply:
Removed.